fmc/src/hand_off.rs

```diff
@@ -44,70 +44,59 @@ impl HandOff {
 
     /// Retrieve FMC CDI
     pub fn fmc_cdi(env: &FmcEnv) -> KeyId {
-        let ds: DataStore = Self::fht(env)
-            .fmc_cdi_kv_hdl
-            .try_into()
-            .unwrap_or_else(|_| {
-                caliptra_common::report_handoff_error_and_halt(
-                    "Invalid CDI DV handle",
-                    caliptra_error::CaliptraError::FMC_HANDOFF_INVALID_PARAM.into(),
-                )
-            });
+        let ds: DataStore =
+            Self::fht(env)
+                .fmc_cdi_kv_hdl
+                .try_into()
+                .unwrap_or_else(|e: CaliptraError| {
+                    cprintln!("Invalid CDI KV handle");
```

Comment:
nit (and similarly for other areas in this file)
```suggestion
                    cprintln!("[fmc] Invalid CDI KV handle");
```

Reply:
Done.